common/archiver/s3store/mocks/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+aws_sdk=$(go list -f '{{.Dir}}' github.com/aws/aws-sdk-go)
+if [ -z "$aws_sdk" ]; then
+  echo "Can't locate aws-sdk-go source code"
```

Comment:
My first idea was to add mitigation instruction here. Like "Run go get -u github.com/aws/aws-sdk-go" but then I thought why not to run it automatically instead of giving error?

Reply:
Turns out `go list` downloads modules as needed!

I don't want to use go get because I explicitly don't want to update versions here, I want to use the version that's already in go.mod